site/content/docs/5.1/components/tags.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+title: Tags
+description: Documentation and examples for tag component.
+group: components
+toc: true
+---
+
+## Standard
+
+{{< example >}}
+<span class="tag">Label</span>
+{{< /example >}}
+
+## With icon
+
+{{< example >}}
+<span class="tag tag-icon">
+
+  <div class="d-flex align-items-center">
+    <svg width="1rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+      <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+    </svg>
+    Label
+  </div>
+</span>
+{{< /example >}}
+
+## With remove icon
+
+{{< example >}}
+<span class="tag tag-remove">
+
+  <div class="d-flex align-items-center">
+    Label
+    <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible remove">
+      <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"/>
+    </svg>
+  </div>
+</span>
+{{< /example >}}
+
+## With icon and remove icon
+
+{{< example >}}
+<span class="tag tag-remove">
+
+  <div class="d-flex align-items-center">
+    <svg width="1rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+      <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+    </svg>
+    Label
+    <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible remove">
+      <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"/>
```

Comment:
I think this svg should be embedded in Boosted directly. It means that this svg should appear in a `::after` class and it should be mentionned in `site/content/docs/5.1/customize/overview.md`.